backend/fms_core/serializers.py

```diff
@@ -783,16 +785,14 @@ def get_steps_order(self, instance):
         return serialized_data.data
 
 class ReferenceGenomeSerializer(serializers.ModelSerializer):
-    editable = serializers.SerializerMethodField()
-    taxon_id = serializers.SerializerMethodField()
+    editable = serializers.BooleanField(read_only=True)
```

Comment:
I might be wrong but i think you need the SerializerMethodField() to call a method for a Field.